examples/hybrid/bubble_2d.jl

```diff
@@ -327,3 +335,175 @@ Plots.png(
     joinpath(path, "energy.png"),
 )
 Plots.png(Plots.plot((Mass .- mass_0) ./ mass_0), joinpath(path, "mass.png"))
+
```

Comment:
I guess we don't need this (from this line to the end)? Or do we need to change this file at all?